cms/envs/common.py

```diff
@@ -83,7 +83,7 @@
 
     # If set to True, Studio won't restrict the set of advanced components
     # to just those pre-approved by edX
-    'ALLOW_ALL_ADVANCED_COMPONENTS': False,
+    'ALLOW_ALL_ADVANCED_COMPONENTS': True,
```

Comment:
@nparlante - we can't turn on all advanced components in studio by default for this bok choy test :)

It actually looks to me like this feature flag is already enabled in the bok_choy environment (see `cms/envs/bok_choy.env.json`). Have you tried running tests without this line?